components/ws-manager-mk2/controllers/subscriber_controller.go

```diff
@@ -76,5 +80,21 @@ func (r *SubscriberReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Ma
 		}
 	}()
 
-	return c.Watch(source.Kind(mgr.GetCache(), &workspacev1.Workspace{}), &handler.EnqueueRequestForObject{})
+	filterByStatus := predicate.Funcs{
```

Comment:
could you add a comment why we're doing this?

Reply:
done